extensions/wikia/PortableInfobox/services/PortableInfoboxRenderService.class.php

```diff
@@ -140,6 +166,26 @@ private function renderInfoboxHero( $data ) {
 	}
 
 	/**
+	 * renders infobox gallery component
+	 *
+	 * @param array $data - array of images
+	 *
+	 * @return string
+	 */
+	private function renderGallery( $data ) {
+		$helper = new PortableInfoboxRenderServiceHelper();
+
+		$galleryData = Array();
+		$hero = array_shift($data);
+		$galleryData['hero'] = $helper->extendImageData( $hero );
```

Comment:
I'm torn as to whether or not that's the best approach. Since mustache doesn't include any logic, you would have to assign a value to the first image in php, something like this:
```php
$galleryData['images'] = [];

foreach ($data as $key => $image) {
	if ($key === 0) {
		$image['first'] = true;
	}
	$galleryData['images'][] = $helper->extendImageData($image);
}
```

and then your mustache template would look something like this:
```mustache
{{#images}}
  {{#first}}
    <figure class="pi-item pi-image pi-image-gallery">
      <a href="{{url}}" class="image image-thumbnail{{#isVideo}} video video-thumbnail small{{/isVideo}}" title="{{alt}}">
        <img src="{{thumbnail}}" class="pi-image-thumbnail" alt="{{alt}}" width="{{{width}}}" height="{{{height}}}" data-{{media-type}}-key="{{name}}" data-{{media-type}}-name="{{name}}"/>
      </a>
      {{#caption}}<figcaption class="pi-item-spacing pi-gallery-caption">{{{caption}}}</figcaption>{{/caption}}
    </figure>
  {{/first}}

  {{^first}}
    <img src="{{thumbnail}}" class="pi-image-thumbnail" alt="{{name}}" data-image-key="{{name}}" data-image-name="{{name}}" style="display: none;" />
  {{/first}}
{{/images}}
```